GetF!t/SelectedTabManager.swift

```diff
@@ -1,13 +1,40 @@
 import Foundation
+import SwiftUI
 
 class SelectedTabManager: ObservableObject {
     @Published var selectedTab = 9
+    @Published var titleText: LocalizedStringKey = LocalizedStringProvider.WelcomePage.welcome
+
+    var exerciseName: () {
+        switch selectedTab {
+        case 0: titleText = LocalizedStringProvider.ExercisesNames.squat
+        case 1: titleText = LocalizedStringProvider.ExercisesNames.stepUp
+        case 2: titleText = LocalizedStringProvider.ExercisesNames.burpee
+        default: titleText = LocalizedStringProvider.ExercisesNames.sunSalute
+        }
+    }
```

Comment:
čo sme si vraveli o properties? Nemá moc zmysel ich použiť keď je datový typ Void. Property si predstav ako nejaké data. Ty v tej property nemáš uložené žiadne data. Ty iba vykonávaš nejakú logiku 😄  Vtedy dáva zmysel to dať ako funkciu. Ideálne by som to nazvala updateExerciseName().

Druhá možnosť je toto zahodiť a ísť napr v goToNextTab cez index (titleText = Exercise.exercises[index].exerciseName)

Reply:
to isté startExerciseButtonTapped a historyButtonTapped v ExerciseViewModel

Reply:
done